### ISSUE TITLE
PushNotification.send warnings about email addresses when device tokens are sent are not helpful

### ISSUE THREAD

**Issue by macMikey** [4d/4D-Mobile-App-Server#13]
If, instead of sending email addresses to **pushNotification.send** I send device tokens, I will receive an error back for every device, saying that I didn't pass an email address.
That is not helpful, since sending device tokens is [specifically mentioned as legitimate in the docs](https://github.com/4d/4D-Mobile-App-Server/blob/main/Documentation/Classes/PushNotification.md#recipients).

**Comment by mesopelagique** [comment]
You are right, if you send device tokens, we must do not care about emails
The error message is?

then did you use this way to pass device tokens
```4d
$recipients:=New object
$recipients.deviceTokens:=New collection("xxxxxxxxxxxx";"yyyyyyyyyyyy";"zzzzzzzzzzzz")

$response:=$pushNotification.send($notification;$recipients)
```
